exercises/bowling/canonical-data.json

```diff
@@ -1,117 +1,110 @@
 {
   "#": [
-    "Bowling is game where players roll a heavy ball to knock down pins",
-    "arranged in a triangle. Write code to keep track of the score of a",
-    "game of bowling."
+    "Students should implement roll and score methods.",
+    "Roll should accept a single integer.",
+    "Score should return the game's final score, when possible",
+    "For brevity the tests all the rolls in an array;",
+    "each element of the rolls array should be passed to the roll method",
+    "The final tests define situations where the score can not be returned",
+    "The expection for these tests is -1, indicating an exception",
```

Comment:
Perhaps `s/exception/error state`. Some languages will implement this as an exception, but others might not (e.g. Go doesn't have exceptions).